lib/rotoscope.rb

```diff
@@ -78,9 +78,9 @@ def inspect
 
   private
 
-  Caller = Struct.new(:entity, :method_name)
-  DEFAULT_CALLER = Rotoscope::Caller.new('<ROOT>', 'unknown').freeze
-  CSV_HEADER_FIELDS = %w(entity method_name method_level filepath lineno caller_entity caller_method_name)
+  Caller = Struct.new(:entity, :method_name, :method_level)
+  DEFAULT_CALLER = Rotoscope::Caller.new('<ROOT>', '<UNKNOWN>', '<UNKNOWN>').freeze
```

Comment:
I've opted to change the default method name from `unknown` to `<UNKNOWN>` to make it clear that it is not a method named `unknown` 😄